### PR TITLE
Splash page

### PR DESCRIPTION
**Pull request by sstrubberg**
https://5cb8e13d5c61c80008c8002e--idr-public.netlify.com/

I'm still working on the cards. The carbon-icons and the svg's from Cris are giving me hard time for some reason.

### CODE REVIEW

File: src/components/Homepage/Homepage.js [hunk 1]
```diff
@@ -60,31 +60,56 @@ export class HomepageHeader extends React.Component {
...
+              aria-label="guiding principles"
+              className="ibm--col-md-4 ibm--col-lg-4 ibm--offset-lg-4 homepage--nav">
+              <div className="homepage--nav-tile">
+                <p>Guiding Principles</p>
+                <p className="bx--type-productive-heading-05">
+                  A foundation for great experiences
+                </p>
+                <Link to="#" alt="Read more">
+                  Read more
+                  <ArrowRight24 aria-label="read more" />
```
Comment (jnm2377) [inline src/components/Homepage/Homepage.js]: It looks like the svg is black so you can't see it on the tile

File: src/components/Homepage/homepage.scss [hunk 2]
```diff
@@ -124,21 +137,24 @@
...
+  @include carbon--breakpoint('md') {
+    padding: 1px !important;
+  }
+}
 
-.container--homepage .ibm--row {
-  position: relative;
-  z-index: 2;
+.homepage--nav-tile {
+  background-color: black;
```
Comment (jnm2377) [inline src/components/Homepage/homepage.scss]: you've been using tokens for other colors, not sure if you meant to use a token here too?

**Comment by jnm2377** [comment]
> The carbon-icons and the svg's from Cris are giving me hard time for some reason.

Is the black icon on the black tile one of those things? Wasn't sure if my comment was one of the things you were having trouble with. It's looking good though!

**Comment by sstrubberg** [comment]
> > The carbon-icons and the svg's from Cris are giving me hard time for some reason.
> 
> Is the black icon on the black tile one of those things? Wasn't sure if my comment was one of the things you were having trouble with. It's looking good though!

yeah, it definitely was. Super helpful too. Take a look :)

**Pull request merged**